site/class-1-play/bjorn2123.svg

```diff
@@ -0,0 +1,137 @@
+<svg xmlns="http://www.w3.org/2000/svg">
```

Comment:
Your `bar` assignment looks great, but it also includes your `play` assignment (which is unfortunately broken).

Could you remove the play assignment from this PR?

P.S. Let me know if I can help.